### PR TITLE
[v0.3] Basic String Sanitizing

### PR DESCRIPTION
**Pull request by matheushf**
- Implement sanitize function

### CODE REVIEW

File: textalyze.js [hunk 1]
```diff
@@ -17,7 +18,33 @@ function itemCounts(array) {
...
+function stringCharsToArray(string) {
+  const arrayString = string.split('');
+  return itemCounts(arrayString);
+}
+
+/**
+ * Takes an arbitrary String as input and returns a "sanitized" string
+ * that replaces all upper-case letters with their lower-case equivalent
+ * @param {String}
+ * @returns {Array}
```
Comment (jfarmer) [inline textalyze.js:44]: Should be `@returns {string}`.

File: textalyze.js [hunk 2]
```diff
@@ -17,7 +18,33 @@ function itemCounts(array) {
...
+ */
+function stringCharsToArray(string) {
+  const arrayString = string.split('');
+  return itemCounts(arrayString);
+}
+
+/**
+ * Takes an arbitrary String as input and returns a "sanitized" string
+ * that replaces all upper-case letters with their lower-case equivalent
+ * @param {String}
```
Comment (jfarmer) [inline textalyze.js:43]: Include descriptions of the parameters in the jsdoc.

File: textalyze.js [hunk 3]
```diff
@@ -17,7 +18,33 @@ function itemCounts(array) {
...
+ * @returns {Array}
+ */
+function sanitize(string = '') {
```
Comment (jfarmer) [inline textalyze.js:46]: I don't think there's a good reason to have a default value for `string`, but if you do have a default value make sure you test it.

**Review by jfarmer — APPROVED** [review]
(Same comments about writing helpful commit messages as other reviews.)

File: textalyze.js (hunk 3 above)
Comment (matheushf) [reply textalyze.js:46]: No me neither, it was just for vscode autocomplete, but I forgot to remove it. Wouldn't happen in a more serious project hehe

File: textalyze.js (hunk 2 above)
Comment (matheushf) [reply textalyze.js:43]: Sorry, forgot about that

File: textalyze.js (hunk 1 above)
Comment (matheushf) [reply textalyze.js:44]: Indeed, I will change it

**Pull request merged**